solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -655,35 +657,11 @@ contract LiquidityProtection is TokenHandler, ContractRegistryClient, Reentrancy
             return (targetAmount, targetAmount, 0);
         }
 
-        // handle base token return
-
-        // calculate the amount of pool tokens required for liquidation
-        // note that the amount is doubled since it's not possible to liquidate one reserve only
-        Fraction memory poolRate = poolTokenRate(liquidity.poolToken, liquidity.reserveToken);
-        uint256 poolAmount = targetAmount.mul(poolRate.d).mul(2).div(poolRate.n);
-
-        // limit the amount of pool tokens by the amount the system/caller holds
-        uint256 availableBalance = store.systemBalance(liquidity.poolToken).add(liquidity.poolAmount);
-        poolAmount = poolAmount > availableBalance ? availableBalance : poolAmount;
+        // calculate the amount of base tokens required for liquidation
```

Comment:
I don't understand this comment - base tokens can't be liquidated, only pool tokens can

Reply:
Can you please fix this comment then?